logging/otel-collector.yaml

```diff
@@ -82,17 +82,11 @@ spec:
               - set(attributes["message"], body)
 
     exporters:
-      loki:
-        endpoint: https://lokistack-dev-gateway-http.openshift-logging.svc.cluster.local:8080/api/logs/v1/application/loki/api/v1/push
-        default_labels_enabled:
-          exporter: true
-          job: true
-        headers:
-          "X-Scope-OrgID": application
+      otlphttp:
+        endpoint: https://lokistack-dev-gateway-http.openshift-logging.svc.cluster.local:8080/api/logs/v1/application/otlp
```

Comment:
Does this work with Loki 5.9? Maybe put a comment here since when it is available.